addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -1028,6 +1028,90 @@ registry.HeaderHamburgerFull = publicWidget.Widget.extend({
     },
 });
 
+registry.BottomFixedElement = publicWidget.Widget.extend({
+    selector: '#wrapwrap',
+    disabledInEditableMode: false,
```

Comment:
I would let it disabled in edit mode, so people can provide the needed spacing should they want to